cmd/environments/get.go

```diff
@@ -61,6 +68,15 @@ func runGet(client environments.Client) func(*cobra.Command, []string) error {
 			return err
 		}
 
+		analyticsTracker.SendEvent(
```

Comment:
This is an example event. We're going to come up with a list of events to track and will implement those in another PR.